### ISSUE TITLE
Plot area does not take big marker sizes into account

### ISSUE THREAD

**Issue by fnorf** [sandialabs/toyplot#164]
I made some circles:

```
import numpy
import toyplot
x = [1, 2]
y = [1, 2]
size = [10, 25]

canvas = toyplot.Canvas()
axes = canvas.cartesian()
mark = axes.scatterplot(x, y, size=size)
```

Seeing the gorgeous examples I somehow expected Toyplot to make sure the charts would always look fancy so I expected the chart area to grow accordingly to the markers.

Instead I got a marker (top right) that is cut off:
![untitled](https://cloud.githubusercontent.com/assets/279583/22656217/f15a08da-ec92-11e6-9b44-f5acac653d97.png)

This is nothing important, probably by design but I wanted to express my surprise. :)

**Comment by tshead2** [comment]
Yeah, this is not intended behavior.  We do explicitly make room for text, we should be doing the same for markers.
